hatchery/openapi/api_workspace_service.go

```diff
@@ -0,0 +1,97 @@
+/*
+ * Hatchery API
+ *
+ * Workspace service for launching and interacting with containers.
+ *
+ * API version: 1.0.0
+ * Generated by: OpenAPI Generator (https://openapi-generator.tech)
+ */
+
+package openapi
+
+import (
+	"context"
+	"net/http"
+	"errors"
+)
+
+// WorkspaceApiService is a service that implements the logic for the WorkspaceApiServicer
+// This service should implement the business logic for every endpoint for the WorkspaceApi API.
+// Include any external packages or services that will be required by this service.
+type WorkspaceApiService struct {
+}
+
+// NewWorkspaceApiService creates a default api service
+func NewWorkspaceApiService() WorkspaceApiServicer {
+	return &WorkspaceApiService{}
+}
+
+// Launch - LaunchAWorkspace
+func (s *WorkspaceApiService) Launch(ctx context.Context, id string, rEMOTEUSER string, authorization string) (ImplResponse, error) {
```

Comment:
Probably not needed. But not really sure how to stop them from being generated. Available options for code generator are at https://openapi-generator.tech/docs/generators/go-server/

Reply:
The current implementation of Hatchery describes an API using an OpenAPI declaration, and then makes several mistakes that break any client that attempts to use the declared API. Using generated code ensures that there are no deviations between the implementation and the declared API specification.

Reply:
okay but why we cannot fix those errors manually by a more intelligent way rather than relying on some auto-generator?

Reply:
Why do something manually when an automated solution exists? The claim that manual solution would be more intelligent is dubious. Once you are using an declarative code generation pattern for API components, updates and changes to protocols become much more manageable.

Reply:
all I'm trying to say here is I don't think introducing a bunch of deadcodes in the attempt to use an automated way to fix this openapi issue is the best way we should move forward with

Reply:
Actually, the file `api_workspace_service.go` can be removed without effecting the build. I can update the Makefile so that it isn't copied after code generation.